src/huggingface_hub/keras_mixin.py

```diff
@@ -16,6 +16,120 @@
 
 logger = logging.getLogger(__name__)
 
+if is_tf_available():
+    import tensorflow as tf
+
+
+def extract_hyperparameters_from_keras(model):
```

Comment:
Should this and other private methods begin with `_` as in `_create_model_card`?

Reply:
yeah I should make them private, thanks! 
I don't think people would ever call extract hyperparameter anyway given they can do it when they have the model